### ISSUE TITLE
G903 not working, G700S is

### ISSUE THREAD

**Issue by patvdleer** [libratbag/libratbag#601]
My old mouse seems to work just fine in Piper, G700S but I just got the new one in, the G903, which doesn't seem to be working...

I've verified that it is listed, one is the receiver, the other is the mouse itself.

```
Bus 005 Device 006: ID 046d:c539 Logitech, Inc. 
Bus 005 Device 005: ID 046d:c086 Logitech, Inc.
```

https://github.com/libratbag/libratbag/blob/master/data/devices/logitech-g903.device

(Ironically enough) I have the following version;
```
libratbag  0.9.903-1-201809262344~ubuntu18.04.1
piper 0.2.902-0-201811051730~ubuntu18.04.1
```

```
$ ratbagctl list -vvv
event20:   Logitech G700s Rechargeable Gaming Mouse
```

**Comment by whot** [comment]
`Bus 005` is bluetooth, does it work with `DeviceMatch=usb:046d:c086;bluetooth:046d:c086;` in the `.device` file?

**Comment by patvdleer** [comment]
That doesn't make any sense when it's connected via usb directly, i.e. without the receiver, and my desktop doesn't have bluetooth on it's own.

Top one is a Dragon Fly USB DAC
```
Bus 005 Device 003: ID 21b4:0081  
Bus 005 Device 010: ID 046d:c086 Logitech, Inc. 
Bus 005 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

While looking into the device file I fixed it, apparently the device file didn't exist.
I've created it at `/usr/share/libratbag/devices/logitech-903.device` according to https://github.com/libratbag/libratbag/blob/master/data/devices/logitech-g903.device
and restarted `ratbagd` via `service ratbagd restart` and it is working now.
